### PR TITLE
No Ticket: Use dashicon for venue span

### PR DESCRIPTION
**Pull request by jmarx**
@mauteri 

Screen Shot 2022-10-29 at 2 48 13 PM

### CODE REVIEW

File: src/components/EventItem.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const EventItem = ( props ) => {
 					</div>
 					{ venue && (
 						<div>
-							Venue: { venue }
+							<span class="dashicons dashicons-location">&nbsp;</span> { venue }
```
Comment (mauteri) [inline src/components/EventItem.js:41]: Take a look at styling done for the dashicons on the editor so you don't need to use `&nbsp;`. I think they are flex layouts if I recall.

Comment (mauteri) [reply src/components/EventItem.js:41]: Also, let's put this after the title rather than before. Thx!

**Review by mauteri — CHANGES_REQUESTED** [review]

File: src/components/EventItem.js [hunk 2]
```diff
@@ -50,7 +50,12 @@ const EventItem = (props) => {
 					>
 						<strong>{event.datetime_start}</strong>
 					</div>
-					{venue && <div>Venue: {venue}</div>}
+					{venue && (
+						<div>
+							{venue}
+							<span className="dashicons dashicons-location"></span>
+						</div>
+					)}
```
Comment (mauteri) [inline src/components/EventItem.js:58]: Let's move this after the title (line 64) so it goes
DATE TIME
TITLE
VENUE

**Review by mauteri — CHANGES_REQUESTED** [review]

File: src/components/EventItem.js (hunk 2 above)
Comment (jmarx) [reply src/components/EventItem.js:63]: @mauteri 
Done

**Review by mauteri — APPROVED** [review]

**Pull request merged**